test/framework/src/main/java/org/elasticsearch/search/aggregations/bucket/AbstractNXYSignificanceHeuristicTestCase.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.search.aggregations.bucket;
+
+import org.elasticsearch.search.aggregations.bucket.terms.heuristic.SignificanceHeuristic;
+
+/**
+ * Abstract test case for testing significant term heuristics
```

Comment:
Maybe a little more detail here?  What makes the NXY version different than plain old `AbstractSignigicanceHeuristicTestCase`.  What does NXY even mean here?

Reply:
I wish I knew what NXY meant, but that is what they are called...Will add some stuff.

Reply:
:lolsob:  Well, at least I'm not the only one.